builder/Dockerfile.alpine-tracee-container

```diff
@@ -2,7 +2,6 @@
 # Creates the official tracee containers.
 #
 
-ARG TARGETARCH
```

Comment:
This should not be global, that's why it's not being accessible inside the stages. See https://docs.docker.com/build/building/variables.